src/xdocs/report_issue.xml

```diff
@@ -113,21 +113,52 @@ Audit done.
 
     </section>
 
-    <section name="How to request new feature for existing functionality?">
+    <section name="How to request enhancement to existing functionality?">
```

Comment:
Why do we need to change the section name? This will break all URLs that point to this.

Reply:
Yes. I'll revert this.

Reply:
Done. @rnveach